netcdfrw2.f

```diff
@@ -1416,7 +1436,7 @@ subroutine netcdfrw2(kopt)
 
       !YuP[2018-09-28], BH181112 added for 'lngshrtf' option,
       !for saving f() distr.func. at selected (nsave()) t steps only.
-      vid=ncvdef2(ncid,'nsave',NCINT,1,tsavedim,istatus)
+      vid=ncvdef2(ncid,'nsave',NCLONG,1,tsavedim,istatus)
```

Comment:
@fmpoli68 , I mentioned to Marina to try fixing this line in her old cql3d reproducing trouble experiments... 🙈